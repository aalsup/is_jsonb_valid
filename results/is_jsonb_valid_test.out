CREATE EXTENSION is_jsonb_valid;
SELECT is_jsonb_valid('{}', '{}');
INFO:  There is no property
 is_jsonb_valid 
----------------
 t
(1 row)

SELECT is_jsonb_valid('{"type": "object"}', '{}');
INFO:  Property
 is_jsonb_valid 
----------------
 t
(1 row)

SELECT is_jsonb_valid('{"type": "object"}', '2');
INFO:  Property
 is_jsonb_valid 
----------------
 t
(1 row)

