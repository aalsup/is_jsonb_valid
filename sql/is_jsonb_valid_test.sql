CREATE EXTENSION is_jsonb_valid;
SELECT is_jsonb_valid('{}', '{}');
SELECT is_jsonb_valid('{"type": "object"}', '{}');
SELECT is_jsonb_valid('{"type": "object"}', '2');