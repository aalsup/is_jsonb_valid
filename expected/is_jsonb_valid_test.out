CREATE EXTENSION is_jsonb_valid;
SELECT is_jsonb_valid('{}', '{}');
 is_jsonb_valid
----------------
 t
(1 row)

SELECT is_jsonb_valid('{"type": "object"}', '{}');
 is_jsonb_valid
----------------
 t
(1 row)

SELECT is_jsonb_valid('{"type": "object"}', '2');
 is_jsonb_valid
----------------
 f
(1 row)

